pages/Zinc-Ingot.md

```diff
@@ -0,0 +1,11 @@
+
+## Obtaining
+Smelting one [Zinc Dust](https://github.com/TheBusyBiscuit/Slimefun4/wiki/Zinc-Dust) in a [Smeltery](https://github.com/TheBusyBiscuit/Slimefun4/wiki/Smeltery) or an [Electric Ingot Factory](https://github.com/TheBusyBiscuit/Slimefun4/wiki/Electric-Ingot-Factory) yields one Zinc Ingot.<br>
+
+Zinc Ingots can be turned back to [Zinc Dust](https://github.com/TheBusyBiscuit/Slimefun4/wiki/Zinc-Dust) by an [Electric Ingot Pulverizer](https://github.com/TheBusyBiscuit/Slimefun4/wiki/Electric-Ingot-Pulverizer)
+
+## Usage
+
+Zinc Ingots are used in the following recipes:
+* [Battery](https://github.com/TheBusyBiscuit/Slimefun4/wiki/Battery)
+* [GPS Teleporter Pylon](https://github.com/TheBusyBiscuit/Slimefun4/wiki/GPS-Teleporter-Pylon)
```

Comment:
Also here: Alloys like Brass

Reply:
Likewise, Brass needs Zinc Dust, not Ingot.
I was thinking about some sort of "overall Dust-Ingot-Alloy diagram" within an article, which show all the relations between them - maybe we could put some "related" links here to help people find what they want?